app/models/journeys/further_education_payments/provider/authorisation.rb

```diff
@@ -2,24 +2,22 @@ module Journeys
   module FurtherEducationPayments
     module Provider
       class Authorisation
-        def initialize(answers:, slug:)
+        def initialize(answers:)
           @answers = answers
-          @slug = slug
         end
 
         def failure_reason
           return :organisation_mismatch if organisation_mismatch?
           return :no_service_access unless answers.dfe_sign_in_service_access?
           return :claim_admin if claim_admin?
           return :incorrect_role unless role_permits_access?
-          return :already_verified if already_verified? && slug != "complete"
```

Comment:
I don't think this is auth related and has now been moved into the normal slug sequence flow